### PR TITLE
feat: Possibility to customize content [Close #29]

### LINKED ISSUE #29: Create the custom template for app
Is it possible to create the custom template for the main page of the application?

### PR DESCRIPTION
**Pull request by picheli20**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
                 <span>-</span>
             </button>
-            <a href="//github.com/ui-jar/ui-jar" target="_blank" class="project-link">GitHub</a>
+            <a
+                [href]="config.project ? config.project.repository : '//github.com/ui-jar/ui-jar'"
+                target="_blank"
+                class="project-link"
+                [innerHTML]="config.project?.repositoryTitle || 'GitHub'"></a>
+        </div>
+        <div>
```
Comment (nording) [inline src/app/app.component.ts]: Is this used (div and content)? I couldn't see it when I was trying - maybe I did something wrong?

File: README.md [hunk 2]
```diff
@@ -127,9 +129,29 @@ You can overwrite the colors using css variables, for example:
...
+
+To overwrite content (f.e. header title, home page, etc) you just need to define the global variable `UIJarConfiguration`, please note that it should be created before angular bootstraps.
+
+One way of doing it is to add `<script>` tag in your `index.html`, like the following example:
+
+```html
+<script>
+    window.UIJarConfiguration = {
+        title: `My custom title`,
+        homePage: `
```
Comment (nording) [inline README.md]: I guess homePage should be homeContent? :)

**Comment by nording** [comment]
@picheli20 nice work! Could you check out my comments and fix that? I will merge then!

File: README.md (hunk 2 above)
Comment (picheli20) [reply README.md]: Ops :p

File: src/app/app.component.ts (hunk 1 above)
Comment (picheli20) [reply src/app/app.component.ts]: Was my bad, was using it to debbug the config injection and see which one is the best solution hehe

**Pull request merged**